### PR TITLE
VMware vSphere Support

### PR DESCRIPTION
**Pull request by holgerkoser**
**What this PR does / why we need it**:
Added VMware `vSphere` as additional infrastructureKind. 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```noteworthy user
Added support for VMware vSphere
```

### CODE REVIEW

File: frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue [hunk 1]
```diff
@@ -96,6 +96,24 @@ limitations under the License.
           ></v-select>
         </v-flex>
       </template>
+      <template v-else-if="infrastructureKind === 'vsphere'">
```
Comment (grolu) [inline frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue]: persistent hint: Default Load Balancer Class is required

File: frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue [hunk 2]
```diff
@@ -324,6 +384,11 @@ export default {
       this.$v.loadBalancerProviderName.$touch()
       this.validateInput()
     },
+    onInputLoadBalancerClassNames () {
+      this.loadBalancerClassNames = intersection(this.allLoadBalancerClassNames, this.loadBalancerClassNames)
```
Comment (grolu) [inline frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue]: comment sorting

File: frontend/src/components/VendorIcon.vue [hunk 3]
```diff
@@ -66,6 +66,10 @@ export default {
           return require('@/assets/digital-ocean.svg')
         case 'vmware':
```
Comment (grolu) [inline frontend/src/components/VendorIcon.vue]: rm vmware if not used

File: frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue (hunk 1 above)
Comment (holgerkoser) [reply frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue]: ✔️

File: frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue (hunk 2 above)
Comment (holgerkoser) [reply frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue]: ✔️

File: frontend/src/components/VendorIcon.vue (hunk 3 above)
Comment (holgerkoser) [reply frontend/src/components/VendorIcon.vue]: ✔️

File: frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue [hunk 4]
```diff
@@ -315,14 +324,16 @@ export default {
...
+      const loadBalancerClasses = map(this.loadBalancerClassesByCloudProfileName(this.cloudProfileName), ({ name, ipPoolName }) => {
         return {
           text: name,
           value: name,
           disabled: name === 'default'
         }
       })
+      return loadBalancerClasses
     },
+
```
Comment (grolu) [inline frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue]:
```suggestion
```

**Pull request merged**